definitions/TalentTab.dbd

```diff
@@ -17,3 +21,61 @@ ClassMask<32>
 PetTalentMask<32>
 OrderIndex<32>
 BackgroundFile
+
+BUILD 4.0.0.12911-4.3.4.15595
+$id$ID<32>
+Name_lang
+SpellIconID<32>
+RaceMask<32>
+CategoryEnumID<32>
+OrderIndex<32>
+BackgroundFile
+Description
+RoleMask<32>
+MasterySpellID<32>[2]
+
+BUILD 4.0.0.11927-4.0.0.12319
+$id$ID<32>
+Name_lang
+SpellIconID<32>
+RaceMask<32>
```

Comment:
Reducing race and class mask should probably leave class mask in talents. 

Also see version above when changing.